MediaBrowser.Common/Net/HttpResponseInfo.cs

```diff
@@ -8,7 +8,7 @@ namespace MediaBrowser.Common.Net
     /// <summary>
     /// Class HttpResponseInfo.
     /// </summary>
-    public class HttpResponseInfo : IDisposable
+    public sealed class HttpResponseInfo : IDisposable
```

Comment:
I think just get rid of the `IDisposable` interface instead, the `Dispose()` method here is empty

Reply:
That would cause a lot of breakage

Reply:
Sorry, I was looking at the error pane in VS wrong when I checked this locally. Yeah that would be a much bigger change.